src/components/Carousel/slides/Swap.js

```diff
@@ -0,0 +1,84 @@
+// @flow
```

Comment:
This file should have been removed entirely

Reply:
it was removed by @machard some weeks ago and reappeared after a rebase.

Reply:
ok. let's ack this. we'll remove on another PR. We have a QA go so i'll merge.